README.md

```diff
@@ -1,87 +1,99 @@
-# Buildkite Test Splitter
+# Buildkite Test Engine Client
 
-Buildkite Test Splitter is an open source tool to orchestrate your test suites. It uses your Buildkite Test Analytics suite data to intelligently partition and parallelise your tests. 
-Test Splitter currently only supports RSpec but support for other frameworks is coming soon.
+Buildkite Test Engine Client (bktec) is an open source tool to orchestrate your test suites. It uses your Buildkite Test Engine suite data to intelligently partition and parallelise your tests.
 
-## Migrating to 0.9.0
-Version 0.9.0 introduces a new feature that requires Test Splitter to read test results from the runner for retries and verification purposes. To enable this feature, it is necessary to configure the `BUILDKITE_SPLITTER_RESULT_PATH` environment variable. This variable specifies the location of where the runner should store test results.
+bktec supports RSpec and Jest.
 
-Furthermore, we have updated the default test command for RSpec to `bundle exec rspec --format progress --format json --out {{resultPath}} {{testExamples}}`. Test Splitter will automatically replace `{{resultPath}}` with the value specified in `BUILDKITE_SPLITTER_RESULT_PATH`. If you want to customize the RSpec command, make sure to include `--format json --out {{resultPath}}` in the command. 
+## Migrating to 1.0.0
```

Comment:
Is `BUILDKITE_TEST_ENGINE_RESULT_PATH` a new mandatory env var in v1.0.0?

Reply:
It became mandatory from 0.9.0
Do we want to highlight it's mandatory in here as well?

Reply:
I think so, If we remove the "Migrating to 0.9.0", some customer could miss that update.

Reply:
It'd be great if `BUILDKITE_TEST_ENGINE_RESULT_PATH` could default to a tempfile or something 🤔

Reply:
Thanks for the feedback. I agree, I think we can make the `BUILDKITE_TEST_ENGINE_RESULT_PATH` optional and default to a temp file. However, you still need to specify the `--format json --out {{resultPath}}` when using custom `BUILDKITE_TEST_ENGINE_TEST_CMD`.